src/FormatNumber/Parser.elm

```diff
@@ -378,3 +380,61 @@ parse locale original =
                 | prefix = locale.zeroPrefix
                 , suffix = locale.zeroSuffix
             }
+
+
+{-| Given a `Locale` parses a `String` into a `Maybe Float`
+-}
+parseString : Locale -> String -> Maybe Float
+parseString locale value =
+    let
+        isNegative : Bool
+        isNegative =
+            negativePrefixMatch && negativeSuffixMatch
+
+        negativePrefixMatch : Bool
+        negativePrefixMatch =
+            value
+                |> String.startsWith locale.negativePrefix
+
+        negativeSuffixMatch : Bool
+        negativeSuffixMatch =
+            value
+                |> String.endsWith locale.negativeSuffix
```

Comment:
I don't think we need either the `|>` syntax for a single function call, and we don't need to make 3 expressions to accomplish what we need:

```suggestion
            String.startsWith locale.negativePrefix value && String.endsWith locale.negativeSuffixMatch value
```

Reply:
Agreed and I have implemented the same.